models/issue_assignees.go

```diff
@@ -171,25 +172,31 @@ func toggleUserAssignee(e *xorm.Session, issue *Issue, assigneeID int64) (remove
 // MakeIDsFromAPIAssigneesToAdd returns an array with all assignee IDs
 func MakeIDsFromAPIAssigneesToAdd(oneAssignee string, multipleAssignees []string) (assigneeIDs []int64, err error) {
 
+	var requestAssignees []string
+
 	// Keeping the old assigning method for compatibility reasons
-	if oneAssignee != "" {
+	if !util.ExistsInSlice(oneAssignee, multipleAssignees) {
+		requestAssignees = append(requestAssignees, oneAssignee)
+	}
 
-		// Prevent double adding assignees
-		var isDouble bool
+	//Prevent empty assignerees
+	if util.ExistsInSlice("", multipleAssignees) {
```

Comment:
My bad! `ExistsInSlice` requires an ordered slice; `IsStringInSlice` performs a linear search.
```suggestion
	if util.IsStringInSlice("", multipleAssignees) {
```